core/src/main/java/org/wildfly/extras/sunstone/api/impl/ec2/EC2Node.java

```diff
@@ -232,34 +235,22 @@ public void stop() {
     public void start() {
         LOGGER.info("Starting {} node '{}'", cloudProvider.getCloudProviderType().getHumanReadableName(), getName());
         LOGGER.debug("Starting instance: {}, instance state: {}", initialNodeMetadata.getId().split("/")[1],  getInstance().getInstanceState());
-        doLifecycle(InstanceState.RUNNING, false); // force (second) parameter does not matter
+        final String timeoutPropertyName = cloudProvider.getProviderSpecificPropertyName(objectProperties,
+                Config.Node.Shared.START_TIMEOUT_SEC);
+        final int timeoutInSec = objectProperties.getPropertyAsInt(timeoutPropertyName, 300);
+        doLifecycle(InstanceState.RUNNING, false, timeoutInSec); // force (second) parameter does not matter
         LOGGER.info("Started {} node '{}'", cloudProvider.getCloudProviderType().getHumanReadableName(), getName());
     }
 
-    /**
-     * Stops the instance. You can also destroy the instance, but that
-     * is something different (the instance cannot be restarted then - it is entirely gone).
-     *
-     * @see org.wildfly.extras.sunstone.api.Node#kill()
-     * @see org.wildfly.extras.sunstone.api.impl.ec2.EC2CloudProvider#destroyNode
-     */
-    @Override
-    public void kill() {
-        LOGGER.info("Killing {} node '{}'", cloudProvider.getCloudProviderType().getHumanReadableName(), getName());
-        LOGGER.debug("Killing instance: {}, instance state: {}", initialNodeMetadata.getId().split("/")[1],  getInstance().getInstanceState());
-        doLifecycle(InstanceState.STOPPED, true);
-        LOGGER.info("Killed {} node '{}'", cloudProvider.getCloudProviderType().getHumanReadableName(), getName());
-    }
```

Comment:
I propose we leave the `kill` method in, I've been looking in the documentation and there actually is a difference between `force = true` and `force = false`.

Here's description of the API call that is made when stopping the instance and the difference the `force` parameter makes: https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_StopInstances.html